libr/core/cmd.c

```diff
@@ -1473,20 +1474,22 @@ static int cmd_kuery(void *data, const char *input) {
 				if (!temp_pos) {
 					break;
 				}
-
 				temp_cmd = r_str_ndup (temp_storage, temp_pos - temp_storage);
-				r_cons_printf ("\"%s\",", temp_cmd);
+				pj_s(pj, temp_cmd);
 				temp_storage += temp_pos - temp_storage + 1;
 			}
-
-			r_cons_printf ("],\n\n");
 			out += cur_pos - out + 1;
 		}
 
-		r_cons_printf ("}}");
+		pj_end(pj);
+		pj_end(pj);
+		pj_end(pj);
+		char *s = pj_drain(pj);
+		r_cons_printf ("%s\" %c\\", s, 10);
```

Comment:
use `r_cons_println (s);` instead, and add a space before every other ( in the above lines